### ISSUE TITLE
convolution.py pad 位置是不是不对？

### ISSUE THREAD

**Issue by LitLeo** [wenet-e2e/wenet#1527]
很小的问题。
原代码
```
if self.lorder > 0:
    if cache is None:
        x = nn.functional.pad(x, (self.lorder, 0), 'constant', 0.0)
    else:
        assert cache.size(0) == x.size(0)
        assert cache.size(1) == x.size(1)
        x = torch.cat((cache, x), dim=2)
    assert (x.size(2) > self.lorder)
    new_cache = x[:, :, -self.lorder:]
else:
    # It's better we just return None if no cache is requried,
    # However, for JIT export, here we just fake one tensor instead of
    # None.
    new_cache = torch.tensor([0.0], dtype=x.dtype, device=x.device)

# GLU mechanism
x = self.pointwise_conv1(x)  # (batch, 2*channel, dim)
x = nn.functional.glu(x, dim=1)  # (batch, channel, dim)

# 1D Depthwise Conv
x = self.depthwise_conv(x)
```

我的理解，pad 是为 depthwise_conv 服务的。但 pointwise_conv1 的 bias 会导致 x 到达 depthwise_conv 时，pad 部分就不是全0了。

经过测试，将pad 挪到pointwise_conv1 后 depthwise_conv 前，对结果几乎没有影响。

但对于 infer 加速来讲，存在一定的影响。这里的infer加速策略是手动实现depthwise_conv，可以在计算过程中按下标直接算pad，实现去除pad操作。但这样逻辑跟python逻辑不同，结果对齐上存在一定的麻烦。

**Comment by kobenaxie** [comment]
Actually yes, padding should move between `point_wise_con1` and `depthwise_conv`, will save some computation. This padding has been changed in [iceall](https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/pruned_transducer_stateless2/conformer.py#L1492) conformer.

**Comment by xingchensong** [comment]
> 

Hi, any numbers on `save some computation` ?
